apps/condo/domains/organization/components/CreateEmployeeForm.tsx

```diff
@@ -1,9 +1,10 @@
-import { Col, Form, Input, Row } from 'antd'
+/** @jsx jsx */
+import { Card, Col, Form, Input, Row } from 'antd'
 import { Rule } from 'rc-field-form/lib/interface'
 import React from 'react'
 import { useOrganization } from '@core/next/organization'
 import { useRouter } from 'next/router'
-import { useIntl } from '@core/next/intl'
```

Comment:
It’s better to use core package to avoid package version conflicts

Reply:
@pahaz you mean react-intl? If so, there will be a lot of changes in all files

Reply:
I can make it in next pr with refactoring